torch_xla/__init__.py

```diff
@@ -42,6 +42,27 @@ def _setup_libtpu_flags():
   # and thus worse performance.
   flags = _set_missing_flags(flags,
                              (('xla_latency_hiding_scheduler_rerun', '1'),))
+
+  if tpu.version() == 5:
+    default_v5_flags = {
+        # Enable async collectives
+        'xla_enable_async_all_gather': 'true',
+        'xla_enable_async_collective_permute': 'true',
+        # Limit compiler-injected rematerialization
+        'xla_jf_rematerialization_percent_shared_memory_limit': '10000',
```

Comment:
Do we need this? I don't think MaxText adds this one as well.

Reply:
For reference: https://github.com/google/maxtext/blob/d2bd833a8b21dd9fd2f02832112dbb4a21b26f69/MaxText/configs/experimental/32b.sh#L32

Reply:
I tried a run without all three flags mentioned, and the absolute MFU reduction was 1.5%. Including this one increases MFU by 1%. I have another run to identify which of the other two flags accounts for the other 0.5% gain.